app/src/pages/inside/projectSettingsPageContainer/content/notifications/ruleGroup/ruleGroup.jsx

```diff
@@ -0,0 +1,335 @@
+import { isEmailIntegrationAvailableSelector } from 'controllers/plugins';
+import {
+  activeProjectRoleSelector,
+  activeProjectSelector,
+  userAccountRoleSelector,
+} from 'controllers/user';
+import Parser from 'html-react-parser';
+import PropTypes from 'prop-types';
+import { withTooltip } from 'components/main/tooltips/tooltip';
+import { TextTooltip } from 'components/main/tooltips/textTooltip';
+import { PROJECT_SETTINGS_TAB_PAGE } from 'controllers/pages';
+import { INTEGRATIONS } from 'common/constants/settingsTabs';
+import { LinkComponent } from 'pages/inside/projectSettingsPageContainer/content/notifications/LinkComponent';
+import arrowRightIcon from 'common/img/arrow-right-inline.svg';
+import { updateNotificationStateAction } from 'controllers/project/actionCreators';
+import { NOTIFICATIONS_PLUGIN_ATTRIBUTE_ENABLED_KEY } from 'controllers/project/constants';
+import { projectPluginNotificationsStateSelector } from 'controllers/project/selectors';
+import { DEFAULT_CASE_CONFIG } from '../constants';
+import { convertNotificationCaseForSubmission } from '../utils';
+import {
+  FieldElement,
+  MODAL_ACTION_TYPE_ADD,
+  MODAL_ACTION_TYPE_COPY,
+  MODAL_ACTION_TYPE_EDIT,
+  NotificationRuleContent,
+  RuleList,
+} from '../../elements';
+import { messages } from '../messages';
+import styles from './ruleGroup.scss';
+import { EmptyRuleState } from '../emptyRuleState';
+
+const cx = classNames.bind(styles);
+const COPY_POSTFIX = '_copy';
+
+const RuleItemDisabledTooltip = withTooltip({
+  TooltipComponent: TextTooltip,
+  data: {
+    width: '265px',
+    placement: 'top',
+    tooltipTriggerClass: cx('info-trigger'),
+    dark: true,
+  },
+})(({ children }) => children);
+
+export const RuleGroup = ({ pluginName, typedRules, notifications }) => {
+  const { trackEvent } = useTracking();
+  const { formatMessage } = useIntl();
+
+  const dispatch = useDispatch();
+  const projectRole = useSelector(activeProjectRoleSelector);
+  const activeProject = useSelector(activeProjectSelector);
+  const userRole = useSelector(userAccountRoleSelector);
+  const isEmailIntegrationAvailable = useSelector(isEmailIntegrationAvailableSelector);
+  const isPluginNotificationsEnabled = useSelector(
+    projectPluginNotificationsStateSelector(pluginName),
+  );
+
+  const isAvailable = () => isEmailIntegrationAvailable && canUpdateSettings(userRole, projectRole);
+
+  const onToggleHandler = (isEnabled, notification) => {
+    trackEvent(PROJECT_SETTINGS_NOTIFICATIONS_EVENTS.SWITCH_NOTIFICATION_RULE(isEnabled));
+
+    dispatch(
+      updateProjectNotificationAction(
+        convertNotificationCaseForSubmission({ ...notification, enabled: isEnabled }),
+      ),
+    );
+  };
+
+  const isReadOnly = !isAvailable();
+
+  const handleRuleItemClick = (isShown) => {
+    if (isShown) {
+      trackEvent(PROJECT_SETTINGS_NOTIFICATIONS_EVENTS.CLICK_TO_EXPAND_NOTIFICATIONS_DETAILS);
+    }
+  };
+
+  const togglePlugin = (isEnabled) => {
```

Comment:
Let's be more specific, as we are not disabling the entire plugin, but its notifications.
e.g. `togglePluginNotificationsEnabled` to be more consistent with `isPluginNotificationsEnabled`.